dev/com.ibm.websphere.appserver.features/visibility/public/messagingClient-3.0/io.openliberty.messagingClient-3.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.messagingClient-3.0
+visibility=public
+singleton=true
```

Comment:
The dependency on private feature com.ibm.websphere.appserver.eeCompatible-9.0 will ensure that messagingClient cannot be configured with any javaEE[6..8]-compatible feature.  Same for the other messaging features.  At server startup when the feature resolver detects such a configuration it will create a singleton conflict.  The feature manager will then log the conflict as an error specific to javaee+jakartaee feature incompatibility and remove the conflicting features from list of features to provision.